src/static/css/footer.less

```diff
@@ -240,7 +240,7 @@
         }
     }
 
-    .list_link__disabled {
```

Comment:
Mm, good catch. This is an artifact of replacing the existing styles. I'll need to check that the links don't need to be a different color here.

As an aside I think this should go back to `u-link__disabled`, at least in the interim. I noticed `footer.less` is filled with `u-link__` patterns, and found it's actually an [established pattern in cf-core](https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-utilities.less#L393-L487).